### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by emilyyyylime**
Found using [typos-cli](https://crates.io/crates/typos-cli) and manually audited by me (a human)

### CODE REVIEW

File: tests/typ/compiler/shorthand.typ [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // Test shorthands for unicode codepoints.
 
 ---
-The non-breaking space~does work, soft-?hypen also does.
+The non-breaking space~does work, soft-?hyphen also does.
```
Comment (PgBiel) [inline tests/typ/compiler/shorthand.typ:4]: Don't forget to update the reference image of this test 🙂

Comment (emilyyyylime) [reply tests/typ/compiler/shorthand.typ:4]: I wasn't sure how I'm supposed to do this. Would I just copy the image over from the generated outputs?

Comment (laurmaedje) [reply tests/typ/compiler/shorthand.typ:4]: The image also should be optimized. You can also use the test CLI for this. See the tests Readme for details.

**Comment by emilyyyylime** [comment]
I understand what went wrong now. I ran the tests after committing, so the updated image wasn't included in the PR. Should be fixed now

**Comment by reknih** [comment]
Thanks!

**Pull request merged**